dump1090.c

```diff
@@ -82,6 +82,7 @@ static void log_with_timestamp(const char *format, ...)
     fprintf(stderr, "%s  %s\n", timebuf, msg);
 }
 
+#ifndef WIN32
 static void sigintHandler(int dummy) {
```

Comment:
@mutability I am thinking about moving both windows and posix implementations to the separate files (i.e. signaling.c). Example of implementation would be like:
posix:
static int* exit_flag = NULL;

static void sigintHandler(int dummy) 
{
 ...
}
void initSignals(int* exit)
{
    exit_flag = exit;
     signal(SIGINT, sigintHandler);
     signal(SIGTERM, sigtermHandler);
}

windows:
static int* exit_flag = NULL;

BOOL WINAPI HandlerRoutine(DWORD dwCtrlType)
{
   ...
}

void initSignals(int* exit)
{
    exit_flag = exit;
   SetConsoleCtrlHandler(HandlerRoutine, TRUE);
}